src/SDKs/Azure.ApplicationModel.Configuration/data-plane/Azure.Configuration.Tests/Readme.md

```diff
@@ -1,16 +1,20 @@
 # Enable Tests against Live Server
 
-1. Get [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest)
+## Prerequisites
+- Get [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest)
+- Have an [Azure subscription](https://docs.microsoft.com/azure/guides/developer/azure-developer-guide#understanding-accounts-subscriptions-and-billing). More information about create a [free account](https://azure.microsoft.com/free/?ref=microsoft.com&utm_source=microsoft.com&utm_medium=docs&utm_campaign=visualstudio).
 
-Run:
+## Usage
+Create or use an existing [Configuration Store](https://docs.microsoft.com/en-us/azure/azure-app-configuration/quickstart-dotnet-core-app#create-an-app-configuration-store).
 
-2. `az login`
-3. `az extension add --source https://azconfigextensions.blob.core.windows.net/azconfigextension/azconfig-0.3.0-py2.py3-none-any.whl`
-4. `az azconfig credential list -n azconfig`
+For users that have access to the `Azure SDK Developer Playground` subscription, run:
```

Comment:
Do we want to keep this here? For external customers this isn't a thing.

Reply:
I thought about that too.... but not sure then where to put the information for internal developers. I do like that as a dev I only go to one place to get all my instructions.
If I don't have access to the `Azure SDK Developer Playground` then I know I have to create my own... no?
Is there a better way to handle this?

Reply:
I think we should move this later in the doc to "contributor" section. The doc should start with information required to simply use the package (without contributing to it).

Reply:
This is in the Tests Readme file though. Are you proposing to add this under a `Contributing` category in here?

Reply:
Ah, never mind then. I thought it's the readme for the component, not for the tests.